src/Sentry.Unity/ScriptableSentryUnityOptions.cs

```diff
@@ -45,7 +45,7 @@ internal static string GetConfigPath(string? notDefaultConfigName = null)
         [field: SerializeField] internal int ShutdownTimeout { get; set; }
         [field: SerializeField] internal int MaxQueueItems { get; set; }
 
-        [field: SerializeField] internal bool IOSNativeSupportEnabled { get; set; }
+        [field: SerializeField] internal bool IosNativeSupportEnabled { get; set; }
```

Comment:
breaking change?

Reply:
Maybe add a new one `bool?` and make this one `[Obsolete]` and read from the old one so we can avoid losing the value when the users upgrade.